gradle/publishing.gradle.kts

```diff
@@ -0,0 +1,49 @@
+apply(plugin = "maven-publish")
+apply(plugin = "signing")
+
+afterEvaluate {
+    configure<PublishingExtension> {
+        publications {
+            create<MavenPublication>("maven") {
+                from(components[componentName()])
```

Comment:
this alone isn't enough for Android

Reply:
See this https://github.com/chrisbanes/gradle-mvn-push/blob/master/gradle-mvn-push.gradle#L89

Reply:
or what this plugin does https://github.com/vanniktech/gradle-maven-publish-plugin

Reply:
Can you have another look? I added generating sources and javadocs jars but I'm still not 100% sure if this is all what's needed for Android.

Reply:
Ok now i see that both jars are empty for Android projects

Reply:
Plot twist: our sources and javadocs have always been empty: https://repo1.maven.org/maven2/io/sentry/sentry-android/4.3.0/